chidioguejiofor/ojumo-website-fend
# Sets up Repo for the frontend

#### What does this PR achieve?
- installs dependencies for the app
- sets up react and react-router using Webpack
- sets up the folder structure of the app
- sets up the React Router for different parts of the app
- adds babel config to the app
- updates README

#### Any background context?
- We intend to use this project to learn the following patterns
   - styled components in JS
   - atomic design

#### Manual Testing Steps
- When you clone the repo
- Run `npm run dev`
- The LandingPage text should be displayed in your browser


#### Any new deployment instructions
- Nay

#### Relevant Trello Tickets
- [Setup Frontend Repo](https://trello.com/c/kbAZ4Siz/2-create-and-setup-frontend-repo)
